_menu.php

```diff
@@ -6,7 +6,8 @@
 
 if($USER->auth>0) {
 	// Logged in
-	$menu->listItem('<a href="publications.php">Publications</a>');
+	$menu->listItem('<a href="pubtrawl.php">Pubtrawl</a>');
```

Comment:
Maybe we should at a later date consider requiring higher levels of authorization (2 or 3) to run pubtrawling or to administer research groups.

This is fine for now

Reply:
Yes, I agree, easy-peasy to fix actually. But I can do that in the next update.
Was also thinking that restricting changing an already verified or discarded publication to admins only.